### ISSUE TITLE
update to Android Lollipop (Material Design style)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [De7vID/klingon-assistant#68]
```
The app needs to be modified to have the latest Material Design styles:
http://www.google.com/design/
http://www.google.com/design/spec/material-design/introduction.html
https://developer.android.com/design/material/index.html

See here for migration how-to:
https://chris.banes.me/2014/10/17/appcompat-v21/
```

Original issue reported on code.google.com by `De.vID.jonpIn` on 25 Dec 2014 at 9:37

**Comment by GoogleCodeExporter** [comment]
```
In particular, currently Android Support Library r20 (support_r20.zip) is 
required.
```

Original comment by `De.vID.jonpIn` on 25 Dec 2014 at 10:01

**Comment by De7vID** [comment]
Fixed in PR #346.
